mmv1/third_party/terraform/tests/resource_composer_environment_test.go.erb

```diff
@@ -1645,7 +1679,7 @@ resource "google_composer_environment" "test" {
 				triggerer {
 					cpu         = 0.5
 					memory_gb   = 2.0
-					count   		= 1
+					count   		= 2
```

Comment:
I suppose it doesn't need to be changed now

Reply:
Agreed, rolling back this line.